blocks/identity-block/themes/commerce.json

```diff
@@ -239,22 +239,27 @@
 	"login-form": {
 		"styles": {
 			"default": {
-				"font-family": "var(--font-family-primary)",
+				"font-family": "Inter",
```

Comment:
@LauraPinilla , looks like this got flipped. Can it use the `var` like `news.json` does?